api/api.proto

```diff
@@ -1373,3 +1457,185 @@ message SdkObjectstoreUpdateRequest {
 message SdkObjectstoreUpdateResponse {
 }
 
+message SdkCloudBackupCreateRequest {
+  // VolumeID of the volume for which cloudbackup is requested
+  string volume_id = 1;
+  // CredentialUUID is cloud credential to be used for backup
+  string credential_uuid = 2;
+  // Full indicates if full backup is desired even though incremental is possible
+  bool full = 3;
+}
+
+message SdkCloudBackupCreateResponse {
+  // Id of the backup created
+  string backup_id = 1;
+}
+
+message SdkCloudBackupRestoreRequest {
+  // Backup ID being restored
+  string backup_id = 1;
+  // Optional volume Name of the new volume to be created
+  // in the cluster for restoring the cloudbackup
+  string restore_volume_name = 2;
+  // The credential to be used for restore operation
+  string credential_uuid = 3;
+  // Optional for provisioning restore
+  // volume (ResoreVolumeName should not be specified)
+  string node_id = 4;
+}
+
+message SdkCloudBackupRestoreResponse {
+  // VolumeID to which the backup is being restored
+  string restore_volume_id = 1;
+}
+
+message SdkCloudBackupDeleteRequest {
+  // ID is the ID of the cloud backup
+  string backup_id = 1;
+  // CredentialUUID is the credential for cloud to be used for the request
+  string credential_uuid = 2;
+  // Force Delete cloudbackup even if there are dependencies
+  bool force = 3;
+}
+
+message SdkCloudBackupDeleteResponse {
+}
+
+message SdkCloudBackupDeleteAllRequest {
+  // id of the volume for the request
+  string src_volume_id = 1;
+  // CredentialUUID is the credential for cloud to be used for the request
+  string credential_uuid = 3;
```

Comment:
is there any specific reason to jump for parameter number here?

Reply:
No, my mistake. I'll fix that.